lib/indico.js

```diff
@@ -17,61 +21,46 @@ var services = [
   'textTags'
 ];
 
-var public_api = 'http://apiv1.indico.io/'
+var public_api = 'http://apiv2.indico.io/'
 
-function service(name, privateCloud) {
+indico.apiKey = false;
```

Comment:
The equivalent of pythons `indicoio.config.apiKey = "YOUR_API_KEY"`.  Allows setting of variables on import.